### PR TITLE
vim-patch:8.0.1394,8.1.1967,8.2.{23,1000,1059,2354,2419,2433,2473,2850}

### PR DESCRIPTION
**Pull request by janlazo**
https://github.com/neovim/neovim/pull/14532/commits/629984a583cee4ed6b7904f758c4f81ff4539d10 doesn't fail in ASAN so the vim patch likely fixed the `tabonly` crash.

### CODE REVIEW

File: src/nvim/globals.h [hunk 1]
```diff
@@ -987,6 +987,8 @@ EXTERN char_u e_dirnotf[] INIT(= N_(
     "E919: Directory not found in '%s': \"%s\""));
 EXTERN char_u e_au_recursive[] INIT(= N_(
     "E952: Autocommand caused recursive behavior"));
+EXTERN char_u e_autocmd_close[] INIT(= N_(
+    "E813: Cannot close autocmd or popup window"));
```
Comment (bfredl) [inline src/nvim/globals.h:991]: only `autocmd window` I think?

File: src/nvim/window.c [hunk 2]
```diff
@@ -6394,6 +6394,10 @@ void restore_win_noblock(win_T *save_curwin, tabpage_T *save_curtab,
   if (win_valid(save_curwin)) {
     curwin = save_curwin;
     curbuf = curwin->w_buffer;
+  } else if (curwin->w_floating) {
```
Comment (bfredl) [inline src/nvim/window.c:6397]: I am not sure this should be done. floating windows can be entered as ordinary windows in neovim, unlike vim popups.

Comment (janlazo) [reply src/nvim/window.c:6397]: I'll consider it N/A then. Up to you to port if needed.

File: src/nvim/globals.h (hunk 1 above)
Comment (janlazo) [reply src/nvim/globals.h:991]: Do we not use this for `floating` windows or do we not make a distinction between floating and regular windows for autocmds?

I don't mind removing `or popup` but I won't update the translations.

Comment (bfredl) [reply src/nvim/globals.h:991]: AFAIK we only use this for autocmd window. Floats can always be closed just like regular windows.

**Pull request merged**